uwblueprint/jump-math
# feat: add archived class indicator

## Notion ticket link

[Add archived class indicator](https://www.notion.so/uwblueprintexecs/Add-archived-class-indicator-cda2368c0b7b46d4b8a2c2996f2a81db?pvs=4)



## Implementation description
* Add archived indicator for archived classes
<img width="1436" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/98d52647-536c-443c-b041-0ff9c8f1f888">




## Steps to test
1. Visit an archived class and verify that the tag exists and looks as expected
2. Visit an active class and verify that the tag does not exist



## What should reviewers focus on?
* 


## Checklist
